ui/components/src/model/units/units.test.ts

```diff
@@ -0,0 +1,110 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { formatValue, UnitOptions } from './units';
+
+describe('formatValue()', () => {
+  const tests = [
+    {
+      value: 100000,
+      unit: { kind: 'Decimal' },
+      expected: '100,000.00',
+    },
+    {
+      value: 155900,
+      unit: { kind: 'Decimal', decimal_places: 4 },
+      expected: '155,900.0000',
+    },
+    {
+      value: 10,
+      unit: { kind: 'Percent' },
+      expected: '10.00%',
+    },
+    {
+      value: 50,
+      unit: { kind: 'Percent', decimal_places: 0 },
+      expected: '50%',
+    },
+    {
+      value: 0.1,
+      unit: { kind: 'PercentDecimal' },
+      expected: '10.00%',
+    },
+    {
+      value: 100,
+      unit: { kind: 'Bytes', decimal_places: 0, abbreviate: false },
+      expected: '100 Bytes',
```

Comment:
I kinda want to lowercase `bytes` :) but not enough to actually go find out what everyone thinks, `Bytes` is good too

Reply:
Checked with Karan and he agrees too, just made this update!